README.md

```diff
@@ -66,25 +76,26 @@ For example, to pin to a specific version of the SDK in your `package.json` file
 }
 ```
 
-## SDK Structure
 
-The `Alchemy` object returned by `new Alchemy()` provides access to the Alchemy API. An optional config
-object can be passed in when initializing to set your API key, change the network, or specify the max number of retries.
+## Alchemy Core
+
+The core package contains all commonly-used [Ethers.js](https://docs.ethers.io/v5/api/providers/api-providers/#AlchemyProvider) methods. If you are already using Ethers.js, you should be simply able to replace the Ethers.js object with `alchemy.core` and it should just work.
 
-There are two different patterns for the Alchemy object to be used:
+It also includes the majority of Alchemy Enhanced APIs, including: 
 
-1. The Alchemy object enables access to methods like `alchemy.getNftsForOwner()` or `alchemy.getBlockNumber()`. This pattern current supports nearly all Ethers.js methods with a 1:1 mapping, the Alchemy NFT API endpoints, and the Alchemy Enhanced APIs.
+- `getTokenMetadata()`: Get the metadata for a token contract address.
+- `getTokenBalances()`: Gets the token balances for an owner given a list of contracts.
+- `getAssetTransfers()`: Get transactions for specific addresses.
+- `getTransactionReceipts()`: Gets all transaction receipts for a given block.
 
-2. In the case that an Ethers.js method is not supported on the top-level, the Alchemy object can be used to generate an Ethers.js provider that allows access to less-common [Ethers.js methods](https://docs.ethers.io/v5/single-page/).
 
-## Accessing the full Ethers.js provider
+### Accessing the full Ethers.js provider
 
 To keep the package clean, we don't support certain uncommonly-used Ethers.js methods as top-level methods the Alchemy object - for example, `provider.formatter`. If you'd like to access these methods, simply use the `alchemy.getProvider()` function to configure the
```

Comment:
can you update this line since it should be alchemy.config.getProvider()

Reply:
done